pegged/parser.d

```diff
@@ -137,18 +137,20 @@ public import pegged.peg;
 import std.algorithm: startsWith;
 import std.functional: toDelegate;
 
+@safe:
+
 struct GenericPegged(TParseTree)
 {
     import std.functional : toDelegate;
     import pegged.dynamic.grammar;
     static import pegged.peg;
-    struct Pegged
+    @safe struct Pegged
     {
     enum name = "Pegged";
-    static ParseTree delegate(ParseTree)[string] before;
-    static ParseTree delegate(ParseTree)[string] after;
-    static ParseTree delegate(ParseTree)[string] rules;
-    static this()
+    static ParseTree delegate(ParseTree) @safe [string] before;
+    static ParseTree delegate(ParseTree) @safe [string] after;
+    static ParseTree delegate(ParseTree) @safe [string] rules;
+    static this() @trusted
     {
```

Comment:
Interesting. Any idea why this can't be `@safe`?

Reply:
toDelegate is @system.